tlog_merge.py

```diff
@@ -7,46 +7,38 @@
 """
 
 from argparse import ArgumentParser
-import os
-import pandas as pd
+
 from pymavlink import mavutil
+
 import table_types
 import util
+from log_merger import LogMerger
 
-# Use MAVLink2
-os.environ['MAVLINK20'] = '1'
 
-
-class TelemetryLogReader:
+class TelemetryLogReader(LogMerger):
     def __init__(self,
-                 tlog_filename: str,
+                 infile: str,
                  msg_types: list[str],
```

Comment:
TIL: I no longer need to import typing.List. Nice!

Reply:
Right, as Python 3.10, I believe.